### DISCUSSION TITLE
PWA cache issue

### DISCUSSION THREAD

**Discussion by vahidtakro** [vercel/next.js#28913] [Help]
I have installed PWA on my next.js app, and there is a problem.

in localhost it works fine,
But deployed on vercel, it first needs to load everything, make a big cache (300mb) then PWA can be installed to devices.
any solution?
maybe I need to disable precache? how can I?

**Comment by FDiskas** [comment]
I would like to suggest you go to directly to plugin discussion. You probably will get more responses
